source/_integrations/lg_thinq.markdown

```diff
@@ -11,6 +11,7 @@ ha_config_flow: true
 ha_domain: lg_thinq
 ha_platforms:
   - switch
+  - binary_sensor
```

Comment:
between Supported devices and Troubleshooting, please add a Section ## Binary sensors listing the names and function of all supported binary sensors